sdk/RELEASE.md

```diff
@@ -7,7 +7,7 @@
 ## Deprecations
 
 ## Bug fixes and other changes
-
+* Fix the compilation error when trying to iterate a list of dictionaries with ParallelFor [\#9366](https://github.com/kubeflow/pipelines/issues/9366)
```

Comment:
```suggestion
* Fix the compilation error when trying to iterate a list of dictionaries with ParallelFor [\#10436](https://github.com/kubeflow/pipelines/pull/10436)
```

Update to point to the PR, rather than the issue.